js/inner-block-slider/index.js

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React, { useState, useRef, useEffect } from 'react';
+
+import { useInnerBlocksProps } from '@wordpress/block-editor';
+import { createBlock } from '@wordpress/blocks';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+import Navigation from './navigation';
+
+/**
+ * InnerBlockSlider component.
+ *
+ * @param {object} root0 - Component props.
+ * @param {string} root0.parentBlockId - Parent block clientId.
+ * @param {number} root0.slidesPerPage - Number of visible slides.
```

Comment:
This is not used any more, only one slide is visible at a time. Should this param be removed?